Resources/Prototypes/_Sunrise/FleshCult/mobs.yml

```
@@ -1,609 +1,609 @@
+      baseSprintSpeed: 5
+    - type: GhostRole
+      allowMovement: true
+      allowSpeech: true
+      makeSentient: true
+      name: mob-flesh-spider-ghost-role-name
+      description: mob-flesh-spider-ghost-role-desc
+      rules: mob-flesh-cult-rules
+      raffle:
+        settings: short
+      mindRoles:
+      - MindRoleGhostRoleTeamAntagonist
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)